docs/conf.py

```diff
@@ -390,6 +390,9 @@
 
 suppress_warnings = ["misc.highlighting_failure"]
 
+# Links we ignore, because they do not work temporary and we cannot fix it
+linkcheck_ignore = ["https://www.virtualbox.org/", "https://www.big-map.eu/"]
```

Comment:
www.virtualbox.org works for me.

Reply:
Yes it hat some down yesterday, not sure what to do. Maybe I just was unlucky, I can remove it again.

Reply:
```suggestion
linkcheck_ignore = ["https://www.big-map.eu/"]
```